langlib/lang.transaction/src/main/ballerina/transaction.bal

```diff
@@ -51,6 +51,16 @@ public type Timestamp readonly & object {
 
 # Returns information about the current transaction.
 #
+# ```ballerina
+# transaction {
+#     transaction:Info info = transaction:info();
+#     check commit;
+# }
```

Comment:
I think we can wrap these in a function since we have another function anyway to show the transactional function case.

Reply:
Do you mean?
```ballerina
public function main() {
    transaction {
        transaction:Info info = transaction:info();
        check commit;
    }
}
```

Reply:
Yeah. We can maybe use a different name instead of main.

Reply:
thannks

Reply:
Yes, applies the same way, right?